docs/strategy-advanced.md

```diff
@@ -229,3 +229,79 @@ for val in self.buy_ema_short.range:
 # Append columns to existing dataframe
 merged_frame = pd.concat(frames, axis=1)
 ```
+
+### Adjust trade position
+
+`adjust_trade_position()` can be used to perform additional orders to manage risk with DCA (Dollar Cost Averaging) for example.
+
+!!! Tip: The `position_adjustment_enable` configuration parameter must be enabled to use adjust_trade_position callback in strategy.
+
+!!! Warning: Additional orders also mean additional fees.
+
+!!! Warning: Stoploss is still calculated from the initial opening price, not averaged price.
```

Comment:
Your syntax for These boxes is off / wrong.

The correct way is the following - with "title" being optional.
you can render the docs with `mkdocs serve` - and have a look at it offline to see if it renders the way you expect it to.
```
!!! <type> "Title"
    <text, indented by 4 spaces>
```